File: Data Structures & Algorithms/maximum-subarray/submission-1.cs
public class Solution {
    public int MaxSubArray(int[] nums) {
        var curSum = 0;
        var maxSum = nums[0];
        foreach (var num in nums)
        {
            curSum = Math.Max(curSum, 0);
            curSum += num;
            maxSum = Math.Max(curSum, maxSum);
        }

        return maxSum;
    }
}

File: Data Structures & Algorithms/max-consecutive-ones/submission-1.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        cpt = 0
        max_cpt = 0
        for value in nums :
            if value == 1 :
                cpt+=1
            else :
                max_cpt = max(cpt, max_cpt)
                cpt = 0
        return max(cpt, max_cpt)